src/Converter.php

```diff
@@ -30,16 +30,22 @@ public function __construct(Currencies $currencies, Exchange $exchange)
     }
 
     /**
-     * @param Money    $money
-     * @param Currency $counterCurrency
-     * @param int      $roundingMode
+     * @param Money             $money
+     * @param Currency          $counterCurrency
+     * @param int               $roundingMode
+     * @param CurrencyPair|null $currencyPair
      *
      * @return Money
      */
-    public function convert(Money $money, Currency $counterCurrency, $roundingMode = Money::ROUND_HALF_UP)
+    public function convert(Money $money, Currency $counterCurrency, $roundingMode = Money::ROUND_HALF_UP, CurrencyPair $currencyPair = null)
```

Comment:
This is a bit problematic, because we would have to check that the currency pair has the same base and counter currencies as the first two arguments. What we really need here is the ratio.

I would probably extract this to a separate function that requires a ratio:

```
// Converts a money to another currency without exchanging a conversion ratio first.
staticConvert(Money $money, Currency $counterCurrency, $ratio, $roundingMode = Money::ROUND_HALF_UP)
```


Or something like that.